app/lib/three_scale/swagger/specification.rb

```diff
@@ -33,9 +33,26 @@ def swagger?
         end
       end
 
+      class V30 < VBase
+        def base_path
+          servers = ThreeScale::OpenApi::UrlResolver.new(@doc).servers
+          servers.first
+        end
 
-      class V20 < VBase
+        JSON_SCHEMA = {'$ref' => 'https://spec.openapis.org/oas/3.0/schema/2019-04-02'}.freeze
+
+        def validate!
+          @validator.fully_validate(JSON_SCHEMA).each do |error|
+            @errors.add(:base, error)
+          end
+        end
 
+        def swagger?
+          true # FIXME: Is it really!?
```

Comment:
I guess so :thinking: The name has changed but it keeps being considered `swagger`, I think :thinking:

Reply:
Yeah. That's weird. I think I won't touch that because it's used only to decide which view template to render in the UI. I'll leave for the UI PR to fix it if needed.